.github/workflows/clean.yml

```diff
@@ -0,0 +1,39 @@
+name: clean
```

Comment:
You may want a different name for either the workflow or job, otherwise you end up with `clean/clean`